roles/setup_repo/tasks/PG_RedHat_setuprepos.yml

```diff
@@ -149,12 +151,33 @@
   failed_when: >
     reposub.rc != 0 or 'error: ' in reposub.stdout.lower()
   changed_when: reposub.rc == '0'
-  when: >
-    tpa_subscription_token|length > 0
-    and install_bdr_packages|bool
-    and pg_version|int >= 14
+  when:
+    - enable_edb_repo|bool
+    - tpa_subscription_token|length > 0
+    - install_bdr_packages|bool
+    - pg_version|int == 14
+    - bdr_version|int == 4
   loop: "{{ edb_2q_repositories }}"
 
+- name: Install PGD packages repo
+  ansible.builtin.yum_repository:
+    name: "enterprisedb-postgres_distributed-{{ item }}"
+    description: "enterprisedb-postgres_distributed-{{ item }}"
+    file: "enterprisedb-postgres_distributed"
+    baseurl: "{{ edb_repo_base_download }}/postgres_distributed/rpm/el/{{ ansible_distribution_major_version }}/{{ item }}"
+    gpgkey: "{{ edb_repo_base_download }}/postgres_distributed/gpg.key"
+    repo_gpgcheck: true
+    enabled: true
+  when:
+    - enable_edb_repo|bool
+    - repo_token|length > 0
+    - install_bdr_packages|bool
+    - pg_version|int >= 15
+    - bdr_version|int >= 5
+  loop:
+    - "x86_64"
+    - "noarch"
+
```

Comment:
Does this same process not need to apply to Debian OS as well? There is a similar `apt_key` and `apt_repository` module to work with

Reply:
I haven't checked Debian yet, but we will add Debian support later. Right now, our focus is on Rocky/RedHat support.